openflow-sdn/openflow-protocol.md

```diff
@@ -0,0 +1,65 @@
+# What is the OpenFlow protocol? 
```

Comment:
Please note that our preferred heading structure is H3 ### and up i.e., H4 ####. Please make the changes where necessary throughout the article in all your headers

For example, your initial header structure ought to be

### What is the OpenFlow protocol? then #### Background on traditional SDN and so on, so forth.